fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkexport/patient/ChunkReader.java

```diff
@@ -133,6 +134,7 @@ protected void fillChunkDataBuffer(List<String> patientIds) throws Exception {
         FHIRSearchContext searchContext;
 
         if (chunkData != null) {
+            patientIds.replaceAll(x -> "Patient/" + x);
```

Comment:
maybe add a TODO with a link to Issue #300 so that we remember to remove this line when we get to that?

Reply:
done